api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -236,6 +236,7 @@ private void updateUserAccessTiers(DbUser dbUser, Agent agent) {
   private class ModuleTimes {
     public Optional<Timestamp> completion;
     public Optional<Timestamp> bypass;
+    final long minExpirationEpochMs = 1625097600000L; // 2021-07-01T00:00:00.000Z
```

Comment:
this should be a static const, so we don't create a new copy of this value for every instance of `ModuleTimes`.

nit: I'd prefer the `Instant.parse()` syntax from the tests, so we don't have to trust that the comment translated epoch-time correctly.